dropshot/src/extractor/body.rs

```diff
@@ -45,6 +45,22 @@ impl<BodyType: JsonSchema + DeserializeOwned + Send + Sync>
     pub fn into_inner(self) -> BodyType {
         self.inner
     }
+
+    pub fn map<T, F>(self, f: F) -> TypedBody<T>
```

Comment:
```suggestion
    /// Convert into a `TypedBody` with a different type parameter; this may be useful when multiple, related endpoint take body parameters that are similar and convertible into a common type.
    pub fn map<T, F>(self, f: F) -> TypedBody<T>
```